compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/ConvertConv2DToIm2ColOp.cpp

```diff
@@ -192,7 +193,7 @@ class ConvertConv2DNhwcHwcf final
   }
 
 private:
-  ControlFnTy controlFn;
+  std::optional<ControlFnTy> controlFn;
```

Comment:
`std::function` can be null -- I don't think we need optional

Reply:
I like to use optional because it makes it clear to me that it is an optional field.